tests/auto/tst_declarativewebcontainer/tst_declarativewebcontainer.cpp

```diff
@@ -229,8 +229,6 @@ void tst_declarativewebcontainer::loading()
     QCOMPARE(m_webContainer->loading(), false);
 
     // Page is not loading => false
-    EXPECT_CALL(page, setWindow(m_webContainer.data()));
```

Comment:
Not per se... Just removed expected calls that are not longer happening. Will amend a comment to the commit log.